tools/config/go-templates/partial_header.mustache

```diff
@@ -1,18 +1,8 @@
 /*
-{{#appName}}
```

Comment:
Our description is very very long and being part of the every file in the SDK it will contribute to the SDK file size. Removing this to make generated code look more close to what humans will write.